.github/docker/docker-compose.yaml

```diff
@@ -45,6 +45,7 @@ services:
       - SCUBA_HEALTHCHECK_FREQUENCY
       - S3QUOTA
       - QUOTA_ENABLE_INFLIGHTS
+      - S3_VERSION_ID_ENCODING_TYPE
```

Comment:
instead of setting this here, should we (can we?) set it automatically in config.js when backend is `mongo` ?

Reply:
I think this should be done in another ticket with a proper analysis